Automattic/themes-workspace
# Implement CSS Variables fallback support

**Description**

Currently this PR adds a postcss plugin that finds all custom properties and turns them into static values, placing a "duplicate" style rule before the custom property.  

**Testing Instructions**

- Check out this PR
- `npm i`
- `npm run build`
- Verify that the styles have not been affected

To-do:
- [x] Add to the package build scripts
- [ ] Test against IE11
- [ ] Explore conditional loading for specific browsers
